src/Microsoft.SqlTools.ServiceLayer/SqlContext/QueryExecutionSettings.cs

```diff
@@ -22,7 +22,7 @@ public class QueryExecutionSettings
         /// <summary>
         /// Default number of chars to store for long text fields (de facto standard as per SSMS)
         /// </summary>
-        private const int DefaultMaxCharsToStore = 65535; // 64 KB - QE default
```

Comment:
I'd recomment keeping this the same as before (it's the max value for an unsigned short) and changing the test instead. This is late in the game and while odds are this won't break something, I'd prefer to be conservative